Doc/Manual/Python.html

```diff
@@ -964,6 +964,7 @@ <H3><a name="Python_commandline">33.2.9 Additional Python commandline options</a
 <tr><td>-nothreads      </td><td>Disable thread support for the entire interface</td></tr>
 <tr><td>-olddefs        </td><td>Keep the old method definitions when using -fastproxy</td></tr>
 <tr><td>-relativeimport </td><td>Use relative Python imports</td></tr>
+<tr><td>-stable-abi     </td><td>Generate code compatible with the Python stable ABI</td></tr>
```

Comment:
I don't really like SWIG's "runallthewordstogether" command line option naming scheme, but `-stableabi` would be more consistent with it.